README.md

```diff
@@ -478,12 +478,15 @@ peaks.on('ready', function() {
 
 Changes the audio or video media source associated with the `Peaks` instance. Depending on the `options` specified, the waveform is either requested from a server or is generated by the browser using the Web Audio API.
 
-The `options` parameter is an object with the following keys. Either `dataUri` or `webAudio` must be specified, but not both.
+The `options` parameter is an object with the following keys. Only one of `dataUri`, or `rawData`, or `webAudio` must be specified.
```

Comment:
I might want to think about the naming here, maybe `waveformData`? But then I'd also want to consider whether the other options (in particular, `dataUri`), should be reorganised.

Reply:
How about `waveformData` and `waveformUri`?

Reply:
We could also maybe just make a single `waveform` or a `preRendered` object and nest `data` and `uri` inside. Or perhaps `local` and `remote`.

Reply:
Or perhaps `remoteWaveform` and `localWaveform`.
How do you think you would like to reorganize things?

Reply:
For now, let's go with `waveformData` instead of `rawData`, for now. I'm thinking of simply renaming `dataUri` to `waveformDataUrl`, but let's do that separately to this PR.